index.mjs

```diff
@@ -100,7 +100,7 @@ function init(open, close) {
 	};
 	return function (txt) {
 		if (this !== void 0 && this.has !== void 0) {
-			this.has.includes(open) || (this.has.push(open),this.keys.push(blk));
+			this.has.indexOf(open) > -1 || (this.has.push(open),this.keys.push(blk));
```

Comment:
The `~` trick is sometimes preferred, I'd be happy to switch that out if you prefer and/or feel free to do so when landing.